redfish-core/lib/account_service.hpp

```diff
@@ -994,6 +996,74 @@ inline void
         std::variant<std::string>(userNameAttribute));
 }
 
+// This file holds the value of the AllowUnauthACFUpload property.  An
+// Administrator user can GET and PATCH this property.  Contrast with the
+// readonly Oem.IBM.ACFWindowActive property (in /redfish/v1) which is true
+// when either AllowUnauthACFUpload or the "OpPanel function 74 time window"
+// is active.  Low-level design:
+//  - When this file exists, AllowUnauthACFUpload=True.
+//  - Otherwise, AllowUnauthACFUpload=False (the default value).
+const std::filesystem::path allowUnauthACFUploadFilename{"/var/lib/AllowUnauthACFUpload"};
```

Comment:
Is it sound for us to be creating this file from bmcweb for the management of this setting? It just seems unconventional since we're normally using dbus getters and setters for properties normally.

Reply:
Correct: this does not follow the pattern of having a D-Bus service.  I looked for a service this setting would go into but did not find one.  And I don't want to create a new one just for this.

Using a file as backing store for a BMC setting is a sound design (AFAIK) and Adriana suggested using /var/lib.  The unconventional element is doing this directly from BMCWeb and not a service.

Reply:
I really think we should follow the architecture of bmcweb and OpenBMC which is Redfish <-> D-Bus